src/ghazal_azadi_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,45 @@
+from random import randint as rand_num
```

Comment:
`from random import randint` is enough.
Using `rand_num` makes it longer and doesn't contribute any new meaning.